smithy-cli/src/main/java/software/amazon/smithy/cli/dependencies/LockFile.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.cli.dependencies;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import software.amazon.smithy.build.model.MavenRepository;
+import software.amazon.smithy.cli.CliError;
+import software.amazon.smithy.model.node.ArrayNode;
+import software.amazon.smithy.model.node.Node;
+import software.amazon.smithy.model.node.ObjectNode;
+import software.amazon.smithy.model.node.StringNode;
+import software.amazon.smithy.model.node.ToNode;
+import software.amazon.smithy.utils.BuilderRef;
+import software.amazon.smithy.utils.ListUtils;
+import software.amazon.smithy.utils.SmithyBuilder;
+import software.amazon.smithy.utils.ToSmithyBuilder;
+
+
+public final class LockFile implements ToSmithyBuilder<LockFile>, ToNode {
```

Comment:
Does this need to be public?

Reply:
If I moved it into the the Command folder then possibly not. I suppose it is only used in the ClasspathAction and LockCommand so that should be OK.